qiskit/pulse/schedule/pulse_schedule.py

```diff
@@ -75,7 +76,7 @@ class Schedule(TimedPulseBlock):
     """Schedule."""
 
     def __init__(self,
```

Comment:
The schedule should still be initialized with raw channels. It should have no notion of a `DeviceSpecification`, but just a list of `Channel`s it has access too. Can optionally be initialized from a classmethod `Schedule.from_device(device_instance)` which internally should call `return Schedule(device_instance.to_channels())`. This way the `Schedule` is JUST a list of commands on channels.